presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -1351,9 +1351,8 @@ private static String castToVarchar(Type type, Object value, FunctionRegistry fu
             return "NULL";
         }
 
-        Signature coercion = functionRegistry.getCoercion(type, VARCHAR);
-
         try {
+            Signature coercion = functionRegistry.getCoercion(type, VARCHAR);
```

Comment:
/s/coersion/coercion

Reply:
What about adding a test case (e.g. SQL based test running "EXPLAIN ..")

Reply:
I repoed the problem by explaining a 1k line query :(